internal/biz/model/resources.go

```diff
@@ -4,24 +4,30 @@ import (
 	"database/sql/driver"
 	"encoding/json"
 	"fmt"
+	"time"
+
 	"github.com/google/uuid"
 	"gorm.io/gorm"
 	"gorm.io/gorm/schema"
-	"time"
 )
 
 type Resource struct {
 	ID           uuid.UUID `gorm:"type:uuid;primarykey"`
+	InventoryId  uuid.UUID `gorm:"index"`
 	OrgId        string    `gorm:"index"`
 	ResourceData JsonObject
 	ResourceType string
 	WorkspaceId  string
```

Comment:
Are you going to move this to Inventory Resource?

Reply:
This field will now live on both the reporter resources and inventory resources. Since we plan to still publish events and workspace updates for the reporter resources, this field needs to remain on the resource model.